### PR TITLE
[receiver/hostmetrics] Add paging utilization metric

### PR DESCRIPTION
**Pull request by cristianciutea**
# Description:

This PR adds system.paging.utilization to hostmetricsreceiver

Link to tracking Issue:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/6221

[Documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/7536/files#diff-5454e93f439411c74bb5c496d6bb05ad7550cb87a9b601ba80127a39a06fec44R14):
New metrics added to Documentation

### CODE REVIEW

File: receiver/hostmetricsreceiver/internal/scraper/pagingscraper/metadata.yaml [hunk 1]
```diff
@@ -46,3 +46,11 @@ metrics:
       aggregation: cumulative
       monotonic: true
     attributes: [type]
+
+  system.paging.utilization:
+    enabled: true
```
Comment (dmitryax) [inline receiver/hostmetricsreceiver/internal/scraper/pagingscraper/metadata.yaml:51]: Please make this metric optional since it's derived from usage

**Comment by dmitryax** [comment]
This metric should be optional and this PR should go once [this issue](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/7139) is resolved

**Comment by dmitryax** [comment]
Please add a changelog entry

File: receiver/hostmetricsreceiver/internal/scraper/pagingscraper/documentation.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ These are the metrics available for this scraper.
 | **system.paging.faults** | The number of page faults. | {faults} | Sum(Int) | <ul> <li>type</li> </ul> |
 | **system.paging.operations** | The number of paging operations. | {operations} | Sum(Int) | <ul> <li>direction</li> <li>type</li> </ul> |
 | **system.paging.usage** | Swap (unix) or pagefile (windows) usage. | By | Sum(Int) | <ul> <li>device</li> <li>state</li> </ul> |
+| system.paging.utilization | Percentage of Swap (unix) or pagefile (windows) utilization. | 1 | Gauge(Double) | <ul> <li>device</li> <li>state</li> </ul> |
```
Comment (dmitryax) [inline receiver/hostmetricsreceiver/internal/scraper/pagingscraper/documentation.md:14]: According to
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md#instrument-naming utilization metrics has to be a fraction of 1, not percentage.

File: receiver/hostmetricsreceiver/internal/scraper/pagingscraper/paging_scraper_windows.go [hunk 3]
```diff
@@ -31,8 +31,10 @@ import (
 )
 
 const (
-	pagingUsageMetricsLen = 1
-	pagingMetricsLen      = 1
+	pagingUsageMetricsLen       = 1
+	pagingUtilizationMetricsLen = 1
```
Comment (dmitryax) [inline receiver/hostmetricsreceiver/internal/scraper/pagingscraper/paging_scraper_windows.go:35]: is this const being used?

File: receiver/hostmetricsreceiver/internal/scraper/pagingscraper/paging_scraper_others.go [hunk 4]
```diff
@@ -32,7 +32,7 @@ import (
 
 const (
 	pagingUsageMetricsLen = 1
-	pagingMetricsLen      = 2
+	pagingMetricsLen      = 3
```
Comment (dmitryax) [inline receiver/hostmetricsreceiver/internal/scraper/pagingscraper/paging_scraper_others.go:35]: I think `pagingUsageMetricsLen` should be increased, not `pagingMetricsLen`

File: receiver/hostmetricsreceiver/internal/scraper/pagingscraper/paging_scraper_windows_test.go [hunk 5]
```diff
@@ -55,8 +57,10 @@ func TestScrape_Errors(t *testing.T) {
 			getPageFileStats: func() ([]*pageFileStats, error) {
 				return []*pageFileStats{testPageFileData}, nil
 			},
-			expectedUsedValue: int64(testPageFileData.usedBytes),
-			expectedFreeValue: int64(testPageFileData.freeBytes),
+			expectedUsedValue:            int64(testPageFileData.usedBytes),
+			expectedFreeValue:            int64(testPageFileData.freeBytes),
+			expectedUtilizationFreeValue: 80.0,
+			expectedUtilizationUsedValue: 20.0,
```
Comment (dmitryax) [inline receiver/hostmetricsreceiver/internal/scraper/pagingscraper/paging_scraper_windows_test.go:63]: Why we have percents here? How does the test pass?

Comment (cristianciutea) [reply receiver/hostmetricsreceiver/internal/scraper/pagingscraper/paging_scraper_windows_test.go]: that's true! in fact I've just seen that the tests are failing when running from my machine. I think they don't get executed in the pipeline.
I've pushed a fix for the test

**Review by dmitryax — APPROVED** [review]

**Pull request merged**